src/js/enactors.js

```diff
@@ -44,7 +44,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     });
 
     gpii.firstDiscovery.enactor.lang.switchLang = function (that) {
-        // Do NOT reload the page at the component instantiation
+        // Do NOT reload the page the first time when the initial language model value is received.
+        // excludeSource: "init" does not help because the language model value is passed in via model
+        // relay. At then, the enactor has been instantiated.
```

Comment:
I think you can remove the last sentence, and instead and the previous sentence as "... via model relay after the model which occurs after "init" takes place."